datasets/flwr_datasets/utils.py

```diff
@@ -25,15 +25,22 @@
 
 tested_datasets = [
     "mnist",
+    "ylecun/mnist",
     "cifar10",
+    "uoft-cs/cifar10",
     "fashion_mnist",
+    "zalando-datasets/fashion_mnist",
     "sasha/dog-food",
     "zh-plus/tiny-imagenet",
     "scikit-learn/adult-census-income",
     "cifar100",
+    "uoft-cs/cifar100",
     "svhn",
+    "ufldl-stanford/svhn",
     "sentiment140",
+    "stanfordnlp/sentiment140",
     "speech_commands",
+    "google/speech_commands",
     "LIUM/tedlium",  # Feature wise it's just like speech_commands
     "flwrlabs/femnist",
```

Comment:
```suggestion
    "flwrlabs/femnist",
    "flwrlabs/ambient-acoustic-context",
    "flwrlabs/ucf101",
    "flwrlabs/shakespeare",
    "flwrlabs/celeba",
```

Reply:
@adam-narozniak should we add the new datasets under `flwrlabs/`?

Reply:
nevermind. let's wait until the other PRs are merged. still i think a couple of dataset will need to be added

Reply:
Thanks for pointing that out, yes a few datasets are still lagging with the tests. They should be added in new PRs. The purpose of this PR is to just handle different namespace of the datasets that already have tests but can be found under different names